src/odemis/gui/cont/correlation.py

```diff
@@ -70,10 +71,23 @@ def __init__(self, tab_data, panel, tab, viewports):
         # disable if no streams are present
         self._tab_data_model.streams.subscribe(self._on_correlation_streams_change, init=True)
         self._tab_data_model.streams.subscribe(self._update_correlation_cmb, init=True)
+        self._tab_data_model.streams.subscribe(self.add_to_localization_tab, init=False)
 
         # connect the correlation streams to the tab data
         self._panel.cmb_correlation_stream.Bind(wx.EVT_COMBOBOX, self._on_selected_stream_change)
 
+        # reference frames
+        self.ref_frames = ["METEOR", "FIBSEM", "No Reference"]
+        self.ref_stream_map = {"METEOR": StaticFluoStream, 
+                               "FIBSEM": StaticSEMStream, 
+                               "No Reference": type(None)} # match everything except None 
+        self.ref_stream = self.ref_stream_map[self.ref_frames[0]]
```

Comment:
Since you are always accessing `self.ref_stream_map` with an index, why not just only have a list? Or have a dict with indices as the keys?

```suggestion
        self.ref_stream_map = [
            StaticFluoStream,  # METEOR
            StaticSEMStream,  # FIBSEM
            type(None)  # match everything except None  (No reference type)
        ]
        self.ref_stream = self.ref_stream_map[0]
```

or
```suggestion
        self.ref_stream_map = {0: StaticFluoStream,  # METEOR
                               1: StaticSEMStream,  # FIBSEM
                               2: type(None)} # match everything except None (No reference type)
        self.ref_stream = self.ref_stream_map[0]
```

Reply:
changed it too a dictionary, but keeping the self.ref_frames (renamed to self.ref_frame_names) because they are used for display